qiskit/providers/fake_provider/__init__.py

```diff
@@ -20,23 +20,27 @@
 The mock devices are mainly for testing the compiler.
 """
 
-from qiskit.test.mock.fake_provider import (
-    FakeProviderForBackendV2,
-    FakeProvider,
-)
-from qiskit.test.mock.fake_provider import FakeProviderFactory
-from qiskit.test.mock.fake_backend import FakeBackend
-from qiskit.test.mock.fake_pulse_backend import FakePulseBackend
-from qiskit.test.mock.fake_qasm_backend import FakeQasmBackend
-from qiskit.test.mock.utils.configurable_backend import ConfigurableFakeBackend
-from qiskit.test.mock.fake_backend_v2 import FakeBackendV2, FakeBackend5QV2
-from qiskit.test.mock.fake_mumbai_v2 import FakeMumbaiFractionalCX
-from qiskit.test.mock.fake_job import FakeJob
-from qiskit.test.mock.fake_qobj import FakeQobj
-
-from qiskit.test.mock.backends import *
-
-from qiskit.test.mock.fake_qasm_simulator import FakeQasmSimulator
-from qiskit.test.mock.fake_openpulse_2q import FakeOpenPulse2Q
-from qiskit.test.mock.fake_openpulse_3q import FakeOpenPulse3Q
-from qiskit.test.mock.fake_1q import Fake1Q
+# Fake job and qobj classes
+from .fake_job import FakeJob
+from .fake_qobj import FakeQobj
+
+# Base classes for fake backends
+from .fake_backend import FakeBackend
```

Comment:
Should I import `FakeBackendV2` here? I am not doing this now because I am trying to avoid the name clash with `fake_backend_v2.FakeBackendV2` mentioned in the comment above. Right now none of these two classes are imported. We can import either one or keep it as is now and address this in the PR for https://github.com/Qiskit/qiskit-terra/issues/7838